### ISSUE TITLE
Uncaught Error: socket hang up

### ISSUE THREAD

**Issue by nisargjhaveri** [atom/atom#5135]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.169.0
**System**: linux 3.14.1-031401-generic
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: socket hang up

```
At events.js:85

Error: socket hang up
  at createHangUpError (_http_client.js:214:15)
  at TLSSocket.socketCloseListener (_http_client.js:246:23)
  at TLSSocket.emit (events.js:129:20)
  at TCP.close (net.js:469:12)

```

### Commands

```
     -31:13.8 editor:fold-current-row (atom-text-editor.editor.is-focused)
  2x -31:13.2 core:move-down (atom-text-editor.editor.is-focused)
  5x -30:51.4 core:backspace (atom-text-editor.editor.mini)
     -30:00.0 pane:show-previous-item (div.settings-view.pane-item)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ]
  },
  "editor": {
    "fontSize": 18,
    "showInvisibles": true,
    "showIndentGuide": true,
    "autoIndentOnPaste": false,
    "tabLength": 4,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v1.1.0
color-picker, v1.3.0
highlight-selected, v0.7.0
project-manager, v1.14.1

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
@kevinsawicki was this one of the errors that was fixed in 0.173.0?

**Comment by Fank** [comment]
yea this were fixed in 0.174 https://github.com/atom/settings-view/commit/3fe0db5dc1c7db9f936b506a126d689057a24e0d
@nisargjhaveri please update to latest atom version

**Comment by nisargjhaveri** [comment]
Thank you. And sorry for an outdated bug report.
